### ISSUE TITLE
Errors when placing startDate, endDate, and ranges on DataRangePicker component

### ISSUE THREAD

**Issue by emgarcia** [skratchdot/react-bootstrap-daterangepicker#84]
The basic code seems to work, but I'm getting errors when I put the startDate, endDate, and ranges properties on the DateRangePicker.

I'm using React 15.2.1.

```
Warning: Unknown props `startDate`, `endDate`, `ranges` on <div> tag. Remove these props from the element.
```

```js
<DateRangePicker
  startDate={this.state.startDate}
  endDate={this.state.endDate}
  ranges={this.state.ranges}
/>
```

**Comment by dkumor** [comment]
It looks like this is fixed, but the fix is not yet on npm. Can the npm package be updated?

**Comment by skratchdot** [comment]
Should be fixed by #85
